test/integration/user_checks_out_from_cart_test.rb

```diff
@@ -35,6 +42,13 @@ class UserChecksOutFromCartTest < ActionDispatch::IntegrationTest
     user = create(:user)
     ApplicationController.any_instance.stubs(:current_user).returns(user)
 
+    StripeMock.start
+    customer = Stripe::Customer.create(
+    {
```

Comment:
Redundant curly braces around a hash parameter.<br>Indent the first parameter one step more than the start of the previous line.